projects/smarti18n/src/lib/utils/string-utils.ts

```diff
@@ -5,19 +5,23 @@ export class StringUtils {
 	 * @param variables Object containing the ```key:value``` pairs to be interpolated.
 	 */
 	public static interpolate(string: string, variables: any): string {
-		const stringVars = string.match(/:\w+[^\s:\.\,$\(\)\[\]\*]/g);
+		if (string) {
```

Comment:
usually when we are talking about a whole function block inside an if, I prefer to do `if (!string) return ''`, so we don't need to waste a full indentation level on basically the whole function (and it looks better too)

Reply:
Makes sense